.circleci/config.yml

```diff
@@ -757,15 +756,10 @@ jobs:
   end-to-end-tests:
     docker:
       - image: circleci/node:12-browsers
-      - image: grafana/grafana-dev:master-$CIRCLE_SHA1
     steps:
-      - attach_workspace:
-          at: /tmp/workspace
       - checkout
-      - run: dockerize -wait tcp://127.0.0.1:3000 -timeout 120s
-      - run:
-          name: CI job started
-          command: ./scripts/ci-job-started.sh
+      - attach_workspace:
+          at: .
```

Comment:
What's the benefit of attaching the workspace in the same directory as opposed to /tmp/workspace? I'm not sure about the ramifications here, but I've generally kept the workspace attached at /tmp/workspace in order to make sure that there aren't conflicts between files in the workspace and in the checked out repo.

Reply:
should be fine & better, I had it a /tmp/workspace at some point but it was not working, could not find the dist/ package files so switch it back to this, can try again